.cargo/config.toml

```
@@ -0,0 +1,2 @@
+[build]
+target = "riscv64imac-unknown-none-elf"
```

```suggestion
[build]
target = "riscv64imac-unknown-none-elf"

```
There should be a new line at the end of each code file, specifically `.toml` and `.rs` files.

---

Thank you for your reminder, I guarantee that I will not commit anymore in the future.